exercises/robot-simulator/src/lib.rs

```diff
@@ -0,0 +1,32 @@
+#[derive(PartialEq, Debug)]
```

Comment:
For my example code I ended up not needing Eq or Copy. But I did require clone. Other students may need different traits, though.

What we might want at the top of all of our stub files is a general comment to say "This file is here to get you started. Change/add/remove code as you see fit."